### PR TITLE
Changing the range value of model IDs on AddCharModel

### PR DESCRIPTION
**Pull request by pushline**

### CODE REVIEW

**Comment by adib-yg** [comment]
If https://github.com/openmultiplayer/open.mp/pull/833 merged, this PR can also be merged.

File: docs/translations/pt-BR/scripting/functions/AddCharModel.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Adiciona um novo modelo de personagem personalizado para download. Os arquivos d
 | Nome    | Descrição                                                                                                                  |
 | ------- | -------------------------------------------------------------------------------------------------------------------------- |
 | baseid  | O ID do modelo da skin original a ser usado como base (personagens originais são usados caso o download falhar).           |
-| newid   | O ID do modelo da nova skin. Varia entre 20.000 a 30000 (10.000 slots) para serem usados posteriormente com SetPlayerSkin. |
+| newid   | O ID do modelo da nova skin. Varia entre 20.001 a 30000 (10000 slots) para serem usados posteriormente com SetPlayerSkin. |
```
Comment (adib-yg) [inline docs/translations/pt-BR/scripting/functions/AddCharModel.md:20]:
```suggestion
| newid   | O ID do modelo da nova skin. Varia entre 20001 a 30000 (10000 slots) para serem usados posteriormente com SetPlayerSkin. |
```

Comment (Y-Less) [reply docs/translations/pt-BR/scripting/functions/AddCharModel.md:20]: `.` is correct for thousands separators in many languages.  The better change suggestion here would be switching `30000` to `30.000`.

Comment (pushline) [reply docs/translations/pt-BR/scripting/functions/AddCharModel.md]: Ive added dots only on english (PT-BR dont use it on numbers >= 1000, only 2 decimals)

**Review by adib-yg — APPROVED** [review]

**Review by adib-yg — APPROVED** [review]

**Pull request merged**